cranelift/filetests/filetests/runtests/arithmetic.clif

```diff
@@ -288,6 +296,8 @@ block0(v0: i16,v1: i16):
 ; run: %udiv_i16(-19, 7) == 0x248F
 ; run: %udiv_i16(0xC0FF, 8) == 0x181F
 ; run: %udiv_i16(0xC0FF, -8) == 0
+; run: %udiv_i16(0x8000, -1) == 0
```

Comment:
I obviously don't understand this change because I would have expected the interpreter to trap in this situation but here it happily returns 0?

Reply:
I guess it's `udiv` vs. `sdiv`? 32768 / 65535 (`-1`) is ~0.5 or 0 rounded down.

Reply:
Yes exactly, `udiv` is not supposed to trap in this situation (only `sdiv`). The -1 is a bit misleading here, because the denominator is interpreted as `0xFFFF`.

Reply:
Ah, that's it, thanks!